hexvexed-0.0.1.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="This is a Hexagon based game for haiku"
```

Comment:
HexVexed is a puzzle game. The objective is to join the different boxes from the right grid in the left grid.